vmdb/app/models/miq_provision_virt_workflow.rb

```diff
@@ -735,21 +735,6 @@ def allowed_snapshots(_options = {})
     result
   end
 
-  def allowed_datastore_storage_controller(_options = {})
-    # TODO: NetAppFiler is an ActAsArModel and doesn't support arel syntax yet
-    NetAppFiler.all.collect(&:name).index_by { |n| n }
-  end
-
-  def allowed_datastore_aggregate(_options = {})
-    result = {}
-    controller = get_value(@values[:new_datastore_storage_controller])
-    return result if controller.blank?
-    naf = NetAppFiler.find_by_name(controller)
-    naf_type = get_value(@values[:new_datastore_fs_type]) == "NFS" ? :aggregates : :volumes
-    naf.send(naf_type).each { |a| result[a] = a }
-    result
-  end
```

Comment:
These two methods cannot be completely remove as users will have existing VM provisioning dialogs that call these methods to populate dialog fields, even if those fields are not actively being used.  For now I would suggest each method just return an empty hash.